src/AppBundle/Controller/Admin/BlogController.php

```diff
@@ -86,7 +87,7 @@ public function newAction(Request $request)
         // However, we explicitly add it to improve code readability.
         // See https://symfony.com/doc/current/best_practices/forms.html#handling-form-submits
         if ($form->isSubmitted() && $form->isValid()) {
-            $post->setSlug($this->get('slugger')->slugify($post->getTitle()));
+            $post->setSlug($slugger->slugify($post->getTitle()));
 
             $entityManager = $this->getDoctrine()->getManager();
```

Comment:
Also we could inject `em` https://github.com/symfony/symfony-docs/pull/7933/files#diff-e530d0758aab398d19d2ed98bf9d6715R107